src/components/tables/AccessTable.less

```diff
@@ -0,0 +1,11 @@
+.access-table .base-table__cell:nth-child(1), 
```

Comment:
can we just do a generic `text-align: left` ? I think there's only two columns

Reply:
I agree!!! 😂